utils/build-windows.bat

```diff
@@ -272,6 +272,7 @@ cmake^
     -DCMAKE_CXX_FLAGS:STRING="/GS- /Oy"^
     -DCMAKE_EXE_LINKER_FLAGS:STRING=/INCREMENTAL:NO^
     -DCMAKE_SHARED_LINKER_FLAGS:STRING=/INCREMENTAL:NO^
+    -DSWIFT_LIT_ARGS="--time-tests"^

```

Comment:
I think that we should retain the `--incremental` default value, even though it would have no bearing.
```suggestion
    -DSWIFT_LIT_ARGS="--incremental --time-tests"^

```